README.md

```diff
@@ -8,11 +8,58 @@ FlowEVM Gateway implements the Ethereum JSON-RPC API for the [FlowEVM](https://d
 
 FlowEVM Gateway is compatible with the majority of standard Ethereum JSON-RPC APIs allowing seamless integration with existing Ethereum-compatible web3 tools via HTTP. FlowEVM Gateway honors Ethereum's JSON-RPC namespace system, grouping RPC methods into categories based on their specific purpose. Each method name is constructed using the namespace, an underscore, and the specific method name in that namespace. For example, the `eth_call` method is located within the `eth` namespace.
 
-Listed below are the JSON-RPC namespaces currently supported by the FlowEVM Gateway:
+Listed below are the JSON-RPC namespaces and methods currently supported by the FlowEVM Gateway:
 
 * `eth`
+  * Supported 
+    * eth_chainId
+    * eth_blockNumber
+    * eth_coinbase
+    * eth_getLogs
+    * eth_getTransactionCount
+    * eth_getTransactionReceipt
+    * eth_getBlockByNumber
+    * eth_call
+    * eth_sendRawTransaction
+    * eth_getTransactionByHash
+    * eth_gasPrice 
+    * eth_getBalance
+    * eth_estimateGas
+    * eth_getTransactionByBlockNumberAndIndex
+    * eth_getTransactionByBlockHashAndIndex
+    * eth_getBlockByHash
+    * eth_getBlockReceipts
+    * eth_getBlockTransactionCountByHash
+    * eth_getBlockTransactionCountByNumber
+    * eth_accounts
```

Comment:
These 4:
- `eth_accounts`
- `eth_sign`
- `eth_signTransaction`
- `eth_sendTransaction`

are also unsupported as described in https://github.com/onflow/flow-evm-gateway/issues/8 . They can be called, but they simply return an error, as described in the issue. So maybe we should move them to unsupported?